src/controller/CHIPDeviceController.cpp

```diff
@@ -675,12 +675,6 @@ CHIP_ERROR DeviceCommissioner::GetDeviceBeingCommissioned(NodeId deviceId, Commi
 CHIP_ERROR DeviceCommissioner::GetConnectedDevice(NodeId deviceId, Callback::Callback<OnDeviceConnected> * onConnection,
                                                   Callback::Callback<OnDeviceConnectionFailure> * onFailure)
 {
-    if (mDeviceBeingCommissioned != nullptr && mDeviceBeingCommissioned->GetDeviceId() == deviceId &&
```

Comment:
I have a strong suspicion that it will break `ChipDeviceCtrl.py` as now it's the controller app that must maintain the "being-commissioned" state to know where to take the device from, and `ChipDeviceCtrl.py` does not seem to do it. Could you verify that BLE pairing using the Python controller still works after this change?

Reply:
Sorry, got a bit sidetracked by something else. Thanks for the good insight - I will try this out this afternoon and fix as appropriate.

Reply:
Added a workaround for chip-device-ctrl in the next commit. I didn't realize the BLE commissioning on python was quite so manual, so good catch. Thanks. For chip-tool, the selection of pase connection is handled internally.